### DISCUSSION TITLE
Gosund SP112 Smart Plug is not detected correctly.

### DISCUSSION THREAD

**Discussion by pietjanssen** [make-all/tuya-local#492] [Q&A] [answered]
Good day,

I'm trying to add my Gosund SP112 Smart plug device to tuya-local, but when I try to add the device, I am presented with a bunch of options, but not the one I'm looking for; Am I missing something? How can I customize my selection? Is it automatically detected? I'm expecting the "Smartplug with USB" to show up.

![image](https://user-images.githubusercontent.com/5436256/222905670-1baa8c9e-bbd2-4f4b-9cf2-4156890c4300.png)

**Comment by make-all** [comment, marked as answer]
Gosund SP112 was added within the last week, so is not in the released version yet.

**Reply by hwinkel** [reply]
I have a Gosund SP112 Called SP112-01 in the IoT Portal, and has the following Product_Id: `zo7k5xd87yciwk6p`

Seems it is not recognized with the latest tuya local version I have: 2023.3.0
